bin/katello-certs-check

```diff
@@ -182,6 +182,18 @@ function check-ca-bundle-size () {
     fi
 }
 
+function check-ca-bundle-trust-rules () {
+    printf "Checking if CA bundle has trust rules: "
+    CHECK=$(awk -v cmd='openssl x509 -noout -text' ' /BEGIN/{close(cmd)};{print | cmd}'  $CA_BUNDLE_FILE |grep 'Trusted Uses'|wc -l)
+    printf $CHECK
+    if [[ $CHECK -lt 1 ]]; then
+                success
```

Comment:
Can you fix the indentation here to align with the rest of the file?

Reply:
I don't see what's wrong with indentation here. What's the problematic line?

Reply:
The file has 4 space indentation and this all lines should be consistent with it.

Reply:
Ack. Thanks for clarifying.